src/requestforattestation/RequestForAttestation.ts

```diff
@@ -109,11 +149,35 @@ export default class RequestForAttestation implements IRequestForAttestation {
     })
   }
 
+  /**
+   * Removes the [[Claim]] Owner from the [[RequestForAttestation]] object.
```

Comment:
Could we briefly explain when/why this is useful (privacy/...)?

Reply:
Again added more context to the sentence